index.js

```diff
@@ -46,6 +56,9 @@ controller.hears(['[\s\S]*'],['direct_message','direct_mention','mention'], func
   // Ping
   if (text === 'ping') return bot.reply('pong');
 
+  // Shortcut for `help(this)`
```

Comment:
As in, new users coming into the channel might want to know about the features, such as the `help()` function (which should probably be `inspect()`), the babel functions, and so on.